frontend/src/components/modules/policy/policyTestMainViewHandler.js

```diff
@@ -288,7 +349,9 @@ const PolicyMainViewHandler = {
 		} = state;
 
 		const showDidYouMean = didYouMean && !loading;
-		const trendingStorage = localStorage.getItem(`trending${cloneData.clone_name}Searches`) || '[]';
+		// const trendingStorage = localStorage.getItem(`trending${cloneData.clone_name}Searches`) || '[]';
```

Comment:
This can be switched back when moving the homepage to the /#/gamechanger.

Reply:
done